app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -1,5 +1,6 @@
 React = require 'react'
 drawingTools = require '../../drawing-tools'
+`import strategies from '../../../features/feedback/shared/strategies';`
```

Comment:
Backticked imports in coffeescript break the tests with `illegal import statement` errors. Use `require(…).default` instead.